layouts/partials/home/use-cases.html

```diff
@@ -0,0 +1,36 @@
+<section class="section has-text-centered use-cases">
+  <div class="container">
+
+    <div class="columns is-variable is-6">
+      <div class="column content is-medium">
+        <p class="image">
+            <a href="#">
+              <img src="/img/usecases/uc1-secure.svg">
+            </a>
+        </p>
+        <p>Secure microservices automatically with Envoy, X.509 PKI or JWT</p>
```

Comment:
```suggestion
        <p>Automatically establish trusted bi-directional communication with Envoy, X.509 PKI or JWT</p>
```

Reply:
I'm open to changes here but it's not clear to me what problem you're trying to solve with this proposal. FWIW this kind of language is generally successful when it speaks to a benefit (eg. "become more secure"), before describing a feature in detail.

I have concerns with the language - "Automatically establish trusted bi-directional communication" as being both jargon-heavy for a home page and opening up unnecessary questions for a new user. What does "establishing communication" mean? Is it a networking tool? Neither SPIFFE nor SPIRE facilitate "communication" per se. Communication between what? users? apps? What does "trusted" mean to a new user to SPIFFE? Trusted by who?

There's prior art for "secure" here as a shorthand establishing strong bi-directional authentication, eg. https://istio.io/  https://aspenmesh.io/

Reply:
I find the term secure to be loose and vague, and I'd rather have a higher degree of specificity on what is being secured, and how is being secured. 

In this case it is the traffic between two microservices as opposed to the microservices directly. 

Perhaps securing microservices communication, or establishing secure microservices connectivity. 

Also considering Evan'ss point, the applicability is not just limited to microservices but many different types of systems.